packages/commercetools/theme/pages/Checkout/OrderReview.vue

```diff
@@ -25,7 +25,7 @@
         <div class="accordion__item">
           <div class="accordion__content">
             <p class="content">
-              <span class="content__label">{{ checkoutGetters.getShippingMethodName(chosenShippingMethod) }}</span><br />
+              <span class="content__label">{{ shippingMethodGetters.getMethodName(chosenShippingMethod) }}</span><br />
```

Comment:
`selectedShippingMethod` instead of `chosenShippingMethod`?

Reply:
It is okay in my opinion in term of naming. However, I am not sure if it is a good idea to change it in this PR, as it comes from useCheckout